scripts/integration-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+INTEGRATION_TEST=$1
+SRC_PATH=$2
+GATSBY_PATH="${TRAVIS_BUILD_DIR:-../../}" # set to third arg if defined, otherwise use ../../
+
+if [[ "$INTEGRATION_TEST" = true ]]; then
+  npm install -g gatsby-dev-cli
```

Comment:
we would need to use `gatsby-dev-cli@next` to get `--copy-all` option

Reply:
Done!